core/src/test/java/org/apache/iceberg/TestSortOrderParser.java

```diff
@@ -47,7 +47,7 @@ public void testUnknownTransforms() {
 
     Assert.assertEquals(10, order.orderId());
     Assert.assertEquals(1, order.fields().size());
-    Assert.assertTrue(order.fields().get(0).transform() instanceof UnknownTransform);
+    org.assertj.core.api.Assertions.assertThat(order.fields().get(0).transform()).isInstanceOf(UnknownTransform.class);
```

Comment:
fqn needed here because https://github.com/apache/iceberg/blob/d4d376bb559e8dc83da3cb7ecbb8ff9205d68f21/core/src/test/java/org/apache/iceberg/TableTestBase.java#L466 is in the same scope